docs/application/dotnet/guides/app-management/data-control.md

```diff
@@ -34,51 +34,58 @@ The main features of the Tizen.Applications.DataControl namespace include:
 
 The data control use cases involve 2 applications. Each application plays a different role: one as the consumer, the other as the provider.
 
-To enable your application to use the data control functionality:
+To enable your application to use the data control functionality, you have to include [Tizen.Applications.DataControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.DataControl.html) namespace.
+```
+using Tizen.Applications.DataControl;
+```
+
+#### Consumer
+Consumer application needs to request specific permission by adding the following privileges to the `tizen-manifest.xml` file:
+```
+<privileges>
+    <privilege>http://tizen.org/privilege/datasharing</privilege>
+    <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+</privileges>
+```
+
+#### Provider
+
+For your privider, you need to include a few things in `tizen-manifest.xml`. If you are using Visual Studio, double-click `tizen-manifest.xml`, and in the manifest editor, go to `Advanced > Data Control`, and click `Add` to add the provider detail. Add the `Read` and `Write` access rights to both `SQL` and `Map` types, as needed.
+
+You can set the data access to be trusted, allowing other applications signed with the same certificate to access the data. You can also define privileges to restrict access to applications having the specific privileges.
+
+If you are not using Visual Studio, this is sample `tizen-manifest.xml`, that you can modify for your needs accordingly.
```

Comment:
If you are not using Visual Studio, ... Is this needed to be mentioned?

Can't we just say:

The following is the sample  this is sample `tizen-manifest.xml`, that you can modify as per your requirement:

Reply:
As I understand, it is because the following code is explaining how to set `Read` and `Write` access rights to both `SQL` and `Map` **without** Visual Studio.

In the line 53, the sentence describes how to do that **with** Visual Studio. By following the guide a user can modify `tizen-manifest.xml` file.
However the user also can manually modify `tizen-manifest.xml` file directly. (With memo, editplus... or something else.) In this case Visual Studio is not needed. So this sentence is to describe how the manually  updated `tizen-manifest.xml` should be.

I think "If you are not using Visual Studio,." is to show this difference.
line 53 will result to the following sample code. You can update the `tizen-manifest.xml` file directly as the following code.
In both ways, the result would be same.

If I understood something wrong, please let me know.

Reply:
@mijin85cho, thanks for clarifying, will wait for @MathewP1 confirmation.

Reply:
@mijin85cho is right. I wanted to give alternative to using Visual Studio by providing fully working `tizen-manifest.xml` and stating that one can use it as a base and modify to their needs.

Reply:
@MathewP1 thanks for confirming, would recommend the following change, also there should be a colon at the end of the sentence (preceding a code sample):

If you are not using Visual Studio, use the following `tizen-manifest.xml` sample and modify it as per your needs:

Reply:
@mijin85cho, agreed, here is a better version.
@MathewP1 sorry for the revised suggestion, could you consider this:

If you are not using Visual Studio, this is sample `tizen-manifest.xml`, that you can modify for your needs accordingly.

to

If you are not using Visual Studio, the following is the `tizen-manifest.xml` sample that you can modify according to your requirement:

Reply:
@MathewP1, you need to fix this one sentence as well.

If you are not using Visual Studio, the following is the tizen-manifest.xml sample that you can modify according to your requirement: